torch/distributed/rpc/__init__.py

```diff
@@ -164,7 +164,7 @@ def init_rpc(
         # keep it alive.
         global rendezvous_iterator
         rendezvous_iterator = dist.rendezvous(
```

Comment:
We might need some renaming here, as `rendezvous()` is no longer a real rendezvous, it just creates a store. And if all we need is a store, we probably should just do that explicitly instead of going through the full `rendezvous` code path.

Reply:
Makes sense, this is not performing rendezvous anymore since nodes are not synchronizing themselves when constructing the store. There is some useful logic in rendezvous that takes `init_method` and validates arguments (`'rank`,`world_size`,`hostname`,`port`) based on the initialization type (`tcp`, `file`, `env`) which I was using before. I created a new method only to perform this validation and create the store `_create_store_for_rpc()` but I need to think about a good way to consolidate the repeated logic.